libphobos/libdruntime/Makefile.am

```diff
@@ -198,11 +199,30 @@ DRUNTIME_DSOURCES_GC = gc/bits.d gc/config.d gc/gc.d gc/os.d \
 
 DRUNTIME_DSOURCES_GCSTUB = gcstub/gc.d
 
-DRUNTIME_DSOURCES_WINDOWS = core/sys/windows/com.d \
-	core/sys/windows/dbghelp.d core/sys/windows/dll.d \
-	core/sys/windows/stacktrace.d core/sys/windows/stat.d \
-	core/sys/windows/threadaux.d core/sys/windows/windows.d \
-	core/sys/windows/winsock2.d
+DRUNTIME_DSOURCES_FREEBSD = core/sys/freebsd/dlfcn.d \
+	core/sys/freebsd/execinfo.d core/sys/freebsd/sys/cdefs.d \
+	core/sys/freebsd/sys/elf.d core/sys/freebsd/sys/elf32.d \
+	core/sys/freebsd/sys/elf64.d core/sys/freebsd/sys/elf_common.d \
+	core/sys/freebsd/sys/event.d core/sys/freebsd/sys/link_elf.d \
+	core/sys/freebsd/sys/mman.d core/sys/freebsd/time.d
+
+DRUNTIME_DSOURCES_LINUX = core/sys/linux/config.d \
```

Comment:
I guess I should add sorting to that script so the order of the `DRUNTIME_DSOURCES_*` does not change on every update ;-)